src/javascripts/ng-admin/Crud/column/maReferencedListColumn.js

```diff
@@ -17,8 +17,9 @@ function maReferencedListColumn(NgAdminConfiguration) {
 <ma-datagrid name="{{ field.datagridName() }}"
     entries="::entries"
     fields="::field.targetFields()"
-    list-actions="::field.listActions()" 
-    entity="::entity">
+    list-actions="::field.listActions()"
+    entity="::entity"
+    datastore="::datastore()">
```

Comment:
Yep, but we'd need to pass an array of ideas and a type to let the datagrid retrieve the entries from the datastore... not ideal.